src/main/java/com/faforever/client/fa/LaunchCommandBuilder.java

```diff
@@ -169,6 +182,17 @@ public LaunchCommandBuilder rehost(boolean rehost) {
       command.add(localIp + ":" + localGpgPort);
```

Comment:
Not part of your changes, but gonna mention it anyways ;P  
`String.format("%s:%d", localIp, localGpgPort)`